openshift/buildConfig-template.yaml

```diff
@@ -7,13 +7,14 @@ metadata:
       This is Thoth User API BuildConfig, this template is meant to be used by
       Bots, but could also be used by humans...
     openshift.io/display-name: "Thoth Core: User API BuildConfig"
-    version: 0.6.0-dev
+    version: 1.0.0
```

Comment:
Not sure if we want to have 1.0.0 now.

Reply:
Ja, let's keep three user-api (and management API?) at the version of the Thoth service: 0.6.0-dev for now, if we think it's good to go to MOC we bounce it to 0.6.0